### PR TITLE
MIN-1549 MIN-1548: LBE V2 and Expired Order Monitor

### PR DESCRIPTION
**Pull request by m1n999999**

### CODE REVIEW

File: src/calculate.ts [hunk 1]
```diff
@@ -1056,6 +1056,8 @@ export namespace StableswapCalculation {
   }
 }
 
+export namespace LbeV2Calculation {}
```
Comment (h2physics) [inline src/calculate.ts:1059]: is it redundant?

File: examples/example.ts [hunk 2]
```diff
@@ -1140,6 +1144,373 @@ async function _cancelStableExample(lucid: Lucid): Promise<TxComplete> {
   });
 }
 
+// MARK: LBE V2
+const ONE_MINUTE_IN_MS = 1000 * 60;
```
Comment (m1n999999) [inline examples/example.ts:1148]: Move to another file

File: src/lbe-v2/lbe-v2.ts [hunk 3]
```diff
@@ -0,0 +1,1885 @@
...
+  if (s1.txHash < s2.txHash) {
+    return -1;
+  }
+  if (s1.txHash === s2.txHash) {
+    return 0;
+  }
+  return 1;
+}
+
+const THREE_HOUR_IN_MS = 3 * 60 * 60 * 1000;
```
Comment (m1n999999) [inline src/lbe-v2/lbe-v2.ts:71]: move to another file

File: src/lbe-v2-worker/main.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
+import { BlockFrostAPI } from "@blockfrost/blockfrost-js";
```
Comment (m1n999999) [inline src/lbe-v2-worker/main.ts:1]: rename file to example

**Review by h2physics — APPROVED** [review]

**Pull request merged**